drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilities.java

```diff
@@ -415,6 +415,10 @@ public String getBoundFactType( String boundName ) {
         return "";
     }
 
+    public GuidedDecisionTable52 getModel() {
```

Comment:
True, I suspect I was lazy here. There is the danger that someday ColumnUtilities loses the model and that causes an unnecessary hit on the analyser codes.